python/amici/ode_export.py

```diff
@@ -1294,6 +1295,13 @@ def _generate_symbol(self, name: str) -> None:
                 if state.conservation_law is None
             ])
             return
+        elif name == 'sx0':
+            self._syms[name] = sp.Matrix([
```

Comment:
Is that used somewhere?

Reply:
I think its just for having more indicative names for the symbols than the default ones, which are currently generated...

Reply:
No it actually had the wrong dimensions. This is theoretically used for `sx0_fixedParameters`, but currently practically not used since the matrix is multiplied with is always zero (which is good since sx0 is missing from the function signature and would thus lead to failures, so this fixes some of the issues but not all).